extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/config/properties/ConfigPropertiesTest.java

```diff
@@ -48,4 +53,21 @@ public void propertiesAffectingSession() {
         assertThat(sessionForOverridesPU.getHibernateFlushMode()).isEqualTo(FlushMode.ALWAYS);
     }
 
+    @Test
+    @Transactional
+    public void extraPhysicalTableTypes() {
+        overrideConfigKey("quarkus.hibernate-orm.\"overridesPU\".database.extra-physical-table-types",
+                "MATERIALIZED VIEW,FOREIGN TABLE");
```

Comment:
This should not be in this method, but next to the other calls to `overrideConfigKey` higher up in this file.

Reply:
You removed the call to `overrideConfigKey`, but didn't add it higher up in this file... ?
As is, the test will just fail.
Can you please fix this and rebase?